### ISSUE TITLE
httokendecode -H sometimes cannot parse htgettoken-obtained token

### ISSUE THREAD

**Issue by shreyb** [fermitools/htgettoken#61]
We're noticing with mu2e's tokens that `httokendecode -H` fails, but `httokendecode` works.  The hangup seems to be that the `human_dates` function in httokendecode is choking on one of the scopes, but it's not clear to me why.  Here are steps to reproduce:

Version of htgettoken:  1.15

```
$ htgettoken -a htvaultprod.fnal.gov -i mu2e
Attempting OIDC authentication with https://htvaultprod.fnal.gov:8200

Complete the authentication at:
    https://cilogon.org/device/?user_code=QLT-HKH-TJK
No web open command defined, please copy/paste the above to any web browser
Waiting for response in web browser
Storing vault token in /tmp/vt_u10610
Saving credkey to /home/sbhat/.config/htgettoken/credkey-mu2e-default
Saving refresh token ... done
Attempting to get token from https://htvaultprod.fnal.gov:8200 ... succeeded
Storing bearer token in /run/user/10610/bt_u10610
$ httokendecode
{
  "wlcg.ver": "1.0",
  "aud": "https://wlcg.cern.ch/jwt/v1/any",
  "sub": <redacted>, 
  "nbf": 1671833600,
  "scope": "storage.create:/mu2e/scratch/users/sbhat storage.read:/mu2e compute.create compute.read compute.cancel compute.modify storage.modify:/mu2e/scratch/users/sbhat",
  "iss": "https://cilogon.org/mu2e",
  "exp": 1671844405,
  "iat": 1671833605,
  "wlcg.groups": [
    "/mu2e"
  ],
  "jti": <redacted>
}
$ httokendecode -H
date: invalid date ‘@storage.read:/mu2e’
parse error: Expected separator between values at line 1, column 303
$
```

This issue was originally raised by @rlcee when testing jobsub_lite, and is reproducible on multiple machines.

Please let me know if you need any more information from me.  Thanks!

**Comment by DrDaveD** [comment]
I believe that's already fixed in 1.16, which is in the osg-testing yum repo at the moment, and the current version in ups/upd.  Please test it.

**Comment by DrDaveD** [comment]
I verified that a mu2e token works with 1.16.
